src/cryptoadvance/specter/device.py

```diff
@@ -124,6 +124,8 @@ def key_types(self, network="main"):
 
     def has_key_types(self, wallet_type, network="main"):
         if wallet_type == "multisig":
+            if self.device_type == "jade":
```

Comment:
I don't think it's a good idea to do jade- specific things in global device class.
You can overwrite `has_key_types` method in jade class.

Reply:
Makes sense, moved it.